.github/workflows/ci-mv3.yml

```diff
@@ -0,0 +1,71 @@
+name: ci-mv3
+on:
+  push:
+    paths:
+      - 'mv3/**'
+  pull_request:
+    paths:
+      - 'mv3/**'
+
+env:
+  XDG_CACHE_HOME: ${{ github.workspace }}/.cache
+
+jobs:
+
+  test:
+    defaults:
+      run:
+        working-directory: ./mv3
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [macos-latest, ubuntu-latest, windows-latest]
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v3
+
+      - name: Install Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.13.x
+
+      - name: Restore .cache
+        uses: actions/cache@v3
+        id: cache
+        with:
+          path: ${{ github.workspace }}/.cache
+          key: ${{ runner.os }}-${{ hashFiles('package*json', 'yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-${{ hashFiles('package*json', 'yarn.lock') }}
+            ${{ runner.os }}-
+
+      - name: Restore node_modules
+        id: yarn-cache
+        uses: actions/cache@v3
+        with:
+          path: node_modules
+          key: ${{ runner.os }}-${{ hashFiles('package*json', 'yarn.lock') }}
```

Comment:
:warning: please  remove `yarn.lock` and this cache – `npm ci` uses `package-lock.json` and ignores `yarn.lock` anyway (which is  good, we want to try using `package-lock.json` and remove `yarn` dependency)